### PR TITLE
Read me

### PR DESCRIPTION
**Pull request by HeroesOfOnesAndZeros**
updated readme to include gif of vno build --ssr
going to recreate the others on ubuntu so they all match

### CODE REVIEW

**Review by deadxears — APPROVED** [review]
weilan approved

File: README.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ _OR_
 deno run --allow-read --allow-write --allow-net --unstable https://deno.land/x/vno/install/vno.ts build --ssr
 ```
 
-![vno build ssr](https://i.ibb.co/q9srX0j/vno-build-ssr.gif)
+![vno build ssr](https://i.ibb.co/41ztNpV/vno-build-ssr.gif
```
Comment (unknownbreaker) [inline README.md]: Will this gif remain on ibb forever, or could it be possible be deleted, at some point?

Comment (unknownbreaker) [reply README.md]: This is OK for now. We can reupload these gifs to have github host them at a later time.

**Review by unknownbreaker — APPROVED** [review]
Approved!

**Pull request merged**